orca-clouddriver/src/test/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/cluster/AbstractClusterWideClouddriverTaskSpec.groovy

```diff
@@ -55,7 +55,7 @@ class AbstractClusterWideClouddriverTaskSpec extends Specification {
       ]
     ])
 
-    (pipeline.stages[0] as AbstractStage).stageNavigator = new StageNavigator(Mock(ApplicationContext))
+    (pipeline.stages[0] as AbstractStage).stageNavigator = new StageNavigator(Stub(ApplicationContext))
```

Comment:
So it turns out that when using `Mock` a call to a method that returns `Map` such as `ApplicationContext.getBeansOfType` will return `null` whereas when using a `Stub` it will return an empty map. Without this change (and the same change in a bunch of other files in this PR) `StageNavigator.stageBuilders` was NPEing all over the place.